Modules/Loadable/Markups/MRML/vtkMRMLMarkupsNode.h

```diff
@@ -252,6 +252,7 @@ class  VTK_SLICER_MARKUPS_MODULE_MRML_EXPORT vtkMRMLMarkupsNode : public vtkMRML
     PointEndInteractionEvent,
     CenterPointModifiedEvent,
     FixedNumberOfControlPointsModifiedEvent,
+    PointAboutToBeRemovedEvent,
```

Comment:
Please add documentation. The proper format for enum documentation is `///<` (instead of `//`). It would be great if you could document the other events, too.